model/registry/profile.yaml

```diff
@@ -0,0 +1,63 @@
+groups:
+
+  - id: registry.profile.frame
+    type: attribute_group
+    brief: >
+      Describes the origin of a single frame in a Profile.
+    attributes:
+      - id: profile.frame.type
+        stability: experimental
+        brief: >
+          Describes the interpreter or compiler of a single frame.
+        examples: ['python']
+        type:
+          allow_custom_values: true
+          members:
+            - id: dotnet
+              brief: >
+                [.Net](https://wikipedia.org/wiki/.NET)
+              value: "dotnet"
+              stability: experimental
+            - id: jvm
+              brief: >
+                [JVM](https://wikipedia.org/wiki/Java_virtual_machine)
+              value: "jvm"
+              stability: experimental
+            - id: kernel
+              brief: >
+                [Kernel](https://wikipedia.org/wiki/Kernel_(operating_system))
+              value: "kernel"
+              stability: experimental
+            - id: native
+              brief: >
+                [C](https://wikipedia.org/wiki/C_(programming_language)),
+                [C++](https://wikipedia.org/wiki/C%2B%2B),
+                [Go](https://wikipedia.org/wiki/Go_(programming_language)),
+                [Rust](https://wikipedia.org/wiki/Rust_(programming_language))
+              value: "native"
+              stability: experimental
+            - id: perl
+              brief: >
+                [Perl](https://wikipedia.org/wiki/Perl)
+              value: "perl"
+              stability: experimental
+            - id: php
+              brief: >
+                [PHP](https://wikipedia.org/wiki/PHP)
+              value: "php"
+              stability: experimental
+            - id: python
+              brief: >
+                [Python](https://wikipedia.org/wiki/Python_(programming_language))
+              value: "python"
+              stability: experimental
+            - id: ruby
+              brief: >
+                [Ruby](https://wikipedia.org/wiki/Ruby_(programming_language))
+              value: "ruby"
+              stability: experimental
+            - id: v8js
+              brief: >
+                [V8JS](https://wikipedia.org/wiki/V8_(JavaScript_engine))
+              value: "v8js"
+              stability: experimental
```

Comment:
This one is a bit of an outlier: all the other frame types here indicate a *language*, not an *implementation*. For example, we have `python` instead of `CPython`, `dotnet` instead of `.NET core`, and `jvm` instead of `hotspot`. V8, on the other hand, is a specific runtime. Should we generalize this to "js"?

Reply:
The change from `v8` to `v8js` originates from a request in https://github.com/open-telemetry/semantic-conventions/pull/1188#discussion_r1658010097 and as https://github.com/open-telemetry/semantic-conventions/pull/1066 did land first, this proposed change did adapt.

Reply:
`v8js` makes sense in the context of what #1066 added because the attributes are actually runtime specific, but this isn't really the case here. 

I can't speak for @trask, but I suspect that his comment was based on the fact that the field already had the runtime-specific name "v8" in your draft. So if we went with that, it'd certainly make sense to go with the already established term `v8js`.

I think both approaches, reporting the general language OR the specific runtime would be raesonable for this field, and I don't have a strong preference. However, we should be *consistent* with that, and not mix it up at random.

Reply:
.NET is not the language but the runtime/framework. You can program in C#, F# and Visual Basic with .NET. Also there's no .NET Core anymore, it's all only .NET.

Reply:
Fair point. The same is also true for JVM: it can also run e.g. Scala, so it's a runtime as well. For most languages in the list, the name of the reference implementation runtime is equal to the language itself. So I think if we'd just rename `Python` to `CPython`, it may already be consistent then.

Reply:
https://github.com/open-telemetry/semantic-conventions/pull/1188/commits/608fb5c84773a3c7d060ebafd351677336e0839b